apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -175,17 +174,14 @@ export default function CourseMap() {
 
     useEffect(() => {
         const updateCalendarEvents = () => {
-            setCalendarEvents(AppStore.getEventsInCalendar());
+            setCalendarEvents(AppStore.getCourseEventsInCalendar());
         };
 
         AppStore.on('addedCoursesChange', updateCalendarEvents);
-        AppStore.on('customEventsChange', updateCalendarEvents);
```

Comment:
Why is this being removed? Everything seems to work fine, so I guess it wasn't necessary but why is that the case?